src/routes/auth.ts

```diff
@@ -0,0 +1,49 @@
+import express from "express"
+import passport from "passport";
+import { SessionOptions } from "passport";
+import { Strategy } from "passport-local";
+import AuthController from "../controllers/authController";
+import { Member, MemberQuery, MemberSchema } from "../types/types";
+import { assert } from "console";
+
+
+const authRouter = express.Router();
+const authController = new AuthController()
+
+passport.use(new Strategy((username, password, done) => {
+  authController.getMember(username).then(
```

Comment:
I know at the moment we are figuring out which fields to use for authentication but uuid is currently a 32 character hash, which seems like a bit of a pain to use for now even for testing, should we change this to something else in the meantime?

Reply:
I think nuid would be more useful in the long run, but for now I'm happy with leaving it as it is.